assets/js/plotter.js

```diff
@@ -363,4 +361,27 @@ function update() {
 	}]);
 	plot.setupGrid();
 	plot.draw();
-}
+} 
+// Pulls the calculated result from the state.dataset array if the user inputs are all numbers
+// @author Chris Want and Gabriella Wahl  
+function gen_output_value () {
+	variables = state.dataset
+	output_value = variables[0][1];
+	for (var i = 0; i < variables.length; i++) {
+		if(variables[i][1] !== variables[0][1]) { 
+			output_value = " "
+			return output_value; 
+		}
+	}
+	return output_value;
+};
+
+function setup_output_interface() {
+	output = output_value;
+	console.log(output_value)
```

Comment:
Oopsie, this logging statement got left in.

Reply:
Oopsies, I will make sure to take that out before I commit the changes!